### PR TITLE
add export to for istio components

### PR DESCRIPTION
**Pull request by linsun**
in case global-default-sidecar-scope.yaml  is applied.

I didn't touch the ingress gateway services.  I'd imagine it needs to be global scoped?

### CODE REVIEW

**Comment by ymesika** [comment]
/lgtm

cc @rshriram

File: install/kubernetes/helm/istio/charts/tracing/templates/service.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ metadata:
     chart: {{ template "tracing.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+  annotations:
+    networking.istio.io/exportTo: "."
```
Comment (rshriram) [inline install/kubernetes/helm/istio/charts/tracing/templates/service.yaml]: this should be set to * as all proxies send data to Zipkin service

Comment (linsun) [reply install/kubernetes/helm/istio/charts/tracing/templates/service.yaml]: updated PTAL @rshriram

**Comment by rshriram** [comment]
cc @duderino it would be nice if this made it in, but not a deal breaker like the other PRs.. This will reduce the number of unnecessary listeners/clusters we setup in all envoys, as well as reduce port conflict issues with other services (e.g., the 9093 port conflict with Kafka that happened recently)

**Comment by costinm** [comment]
/hold
Too little testing - and it will interfere with any effort to run in multiple namespaces or other custom configs.

**Comment by linsun** [comment]
@costinm I added on env WG agenda to discuss custom install regarding namespaces.  We need to get the common pattern captured like the profiles we produced so we can handle the common scenarios first.

**Comment by linsun** [comment]
@costinm 1.1 is shipped - can this be merged now?

**Comment by costinm** [comment]
Is there any user bug about this for 1.1 ? If not - master please, and given Code Mauve please make sure there is at least a test to verify it does whatever it's supposed to do ( and I'm not entirely sure what will be the impact from a user perspective ).

Also please remember that in 1.2 we intended to use the a-la-carte/modular installer, so this change will only affect bookinfo and demo mode - does a demo really need this ?

**Comment by linsun** [comment]
user reported bug:  https://github.com/istio/istio/issues/12237

**Comment by duderino** [comment]
@linsun, I think @nrjpoddar is not a user but rather a provider.  We need to become increasingly risk adverse for 1.1 and instead focus on master and 1.2....

So... I think either we need to address this with a doc fix or we need to test it more thoroughly / explain how it has already been tested more thoroughly

CC @costinm

**Comment by duderino** [comment]
@linsun I think this should go to master and release-1.2 if it's not already there.  Sorry this didn't make it into 1.1

**Pull request closed without merging**